--- ../ares/ares/ps1/spu/reverb.cpp	2026-09-19 00:43:27.595955300 -0700
+++ spu/reverb.cpp	2026-09-19 10:49:57.175879400 -0700
@@ -94,10 +94,10 @@
 
 auto SPU::Reverb::read(u32 virtualAddress, s32 offset) -> s16 {
   u32 realAddress = address((virtualAddress << 2) + offset);
-  return spu.ram.readHalf(realAddress);
+  return self.ram.readHalf(realAddress);
 }
 
 auto SPU::Reverb::write(u32 virtualAddress, s16 data) -> void {
   u32 realAddress = address(virtualAddress << 2);
-  return spu.ram.writeHalf(realAddress, data);
+  return self.ram.writeHalf(realAddress, data);
 }
